### PR TITLE
PHOENIX-7625 Adding query plan information to connection Activity Logger

### PR DESCRIPTION
**Pull request by vmeka2020**

### CODE REVIEW

**Comment by vmeka2020** [comment]
Link to CI build: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-2246/

File: phoenix-core/src/it/java/org/apache/phoenix/jdbc/LoggingConnectionLimiterIT.java [hunk 1]
```diff
@@ -186,6 +187,55 @@ public void testActivityLogsOnQueryWhenFailures() throws Exception {
...
+
+    if (getConnection() instanceof PhoenixConnection) {
+      try (PhoenixConnection pconn = getConnection().unwrap(PhoenixConnection.class);
+        Statement stmt = pconn.createStatement()) {
+        ResultSet rs = stmt.executeQuery(query);
+        while (rs.next()) {
+          // do nothing
+        }
+        boolean queryPlanFound = false;
+        String queryPlan = pconn.getActivityLogger().getExplainPlanInfo();
```
Comment (virajjasani) [inline phoenix-core/src/it/java/org/apache/phoenix/jdbc/LoggingConnectionLimiterIT.java:203]: Can we add a test in some other IT class where we can validate that queryPlan is null by default?

**Review by virajjasani — COMMENTED** [review]
Left one minor comment, looks good otherwise

File: phoenix-core/src/it/java/org/apache/phoenix/jdbc/LoggingConnectionLimiterIT.java (hunk 1 above)
Comment (vmeka2020) [reply phoenix-core/src/it/java/org/apache/phoenix/jdbc/LoggingConnectionLimiterIT.java:203]: @virajjasani added the UT

**Review by virajjasani — APPROVED** [review]
+1

**Comment by virajjasani** [comment]
Awaiting final build results before merging the PR
